### ISSUE TITLE
PINCE sometimes terminates it's inferior when detached

### ISSUE THREAD

**Issue by korcankaraokcu** [korcankaraokcu/PINCE#1]
This bug is probably related to gdb/mi

Edit: The thing that causes this problem is gdb/mi's `set target-async 1` command. If the target doesn't support asynchronous execution, this problem occurs. Might delete `set target-async 1` command and redesign the PINCE to cover the lack of it. I really don't want to leave it as it is.

**Comment by korcankaraokcu** [comment]
I realized that it's impossible to redesign PINCE without the `set target-async 1` command. Without it, it'd be impossible to switch threads fast and efficiently. So even if I don't want to leave it as it is, seems like I kinda have to. Most of the programs doesn't get affected anyways. Only program that closed itself is firefox and it only closes itself after watching a video. I tried to attach and detach like at least 20-25 times to Torchlight II and nothing happened. So, closing this issue since there can't be anything done with it. Hope they will fix the serious bugs in gdb/mi.
